packages/Webkul/Installer/src/Console/Commands/Installer.php

```diff
@@ -92,7 +94,7 @@ protected function createEnvFile()
             $this->info('Your Default Timezone is ' . date_default_timezone_get());
 
             // Updating App Default Currencies
-            $currencies = ['USD', 'EUR'];
+            $currencies = ['CNY', 'AED', 'EUR', 'INR', 'IRR', 'ILS', 'JPY', 'GBP', 'RUB', 'SAR', 'TRY', 'USD', 'UAH'];
```

Comment:
You will also need to seed the currencies to the database, currently only **EUR** and **USD** exist.

![image](https://github.com/bagisto/bagisto/assets/1594411/35af5a8a-e431-4575-9415-9013dcba5eda)

Reply:
Only selected currency during the installation will be created. We will add a feature to create supported currencies and locales in the future.